pkg/controller/direct/logging/utils.go

```diff
@@ -130,7 +192,16 @@ func convertAPItoKRM_LogMetricLabels(apiLabels []*api.LabelDescriptor) []krm.Log
 			Key:         &apiLabel.Key,
 			ValueType:   &apiLabel.ValueType,
 		}
+		// TODO(acpana): this looks like a bug on the api, as the value should be defaulted already.
+		if *kccLabels[i].ValueType == "" {
```

Comment:
Nit: It's more robust to use ValueOf, I know you set it two lines above, but the problem is that when/if you refactor it's easy to introduce bugs that way.